web/styles/rendered_markdown.css

```diff
@@ -411,35 +411,12 @@
 
     .twitter-image,
     .message_inline_image {
-        position: relative;
+        display: inline-block;
+        vertical-align: middle;
+
         margin-bottom: var(--markdown-interelement-space-px);
         margin-right: 5px;
 
-        /* Sizing CSS for inline images requires care, because images load
-           asynchronously, and browsers will unfortunately jump your
-           scroll position when elements load above the current
-           position in the message feed in a way that changes the
-           height of elements. (As of March 2022, both Firefox and
-           Chrome exhibit this problem, though in Chrome it is pretty
-           subtle).
```

Comment:
I think this context comment still belongs here somewhere, probably just at the top of the `img` block.

Reply:
Agreed. Placed a rewritten version of this.